gestalt-util/src/main/java/org/terasology/gestalt/util/collection/CircularDependencyException.java

```diff
@@ -21,21 +21,44 @@
  * and thus no linear ordering exists.
  */
 public class CircularDependencyException extends RuntimeException {
+    /**
+     * the exception
```

Comment:
```suggestion
     * Constructs a CircularDependencyException.
```
I am not sure what purpose JavaDoc serves on an empty constructor of a documented class, however, if the linter wishes it then we can do that.